osu.Game/Online/Rooms/MultiplayerPlaylistItem.cs

```diff
@@ -69,6 +72,7 @@ public MultiplayerPlaylistItem(PlaylistItem item)
             Expired = item.Expired;
             PlaylistOrder = item.PlaylistOrder ?? 0;
             PlayedAt = item.PlayedAt;
+            StarRating = item.Beatmap.StarRating; // generally not available, but lets at least try to use it
```

Comment:
It is available only if passed in `PlaylistItem` was previously created with `MultiplayerPlaylistItem` which was fetched from the server. I believe this line is the only change that is not required for what this PR tries to achieve, but i thought it would be good to copy it if it is there. The reason i added this comment is [this](https://github.com/ppy/osu/blob/master/osu.Game/Online/Rooms/PlaylistItem.cs#L72).

Reply:
Looking at usages this ctor seems to be mostly used in tests.

I'd remove the comment though because it's vague to the point of being almost negative information. The md5 hash copy a few lines higher does not have it despite being susceptible to the same problem. Or, I guess if we want to keep it for whatever reason, it should point at the `PlaylistItem.Beatmap` xmldoc specifically. But xmldoc'ing this entire ctor as "for test purposes only" would really be the ideal outcome for me here because otherwise this ctor suggests client->server data flow which does not ever happen that way (I would hope).

Reply:
done